public/scripts/client/subcomponents/PingForm.jsx

```diff
@@ -48,8 +42,17 @@ var PingForm = React.createClass({
   render: function() {
     return(
```

Comment:
Need to add this before returning the html:

`var member = this.props.members.reduce(function(a,b) {
  return a || (b.id === id ? b : null);
}, null);`